go.mod

```diff
@@ -2,8 +2,12 @@ module github.com/rancher/rancher
 
 go 1.20
 
-// on release remove this wrangler replace and use the latest tag
-replace github.com/rancher/wrangler v1.1.1 => github.com/rancher/wrangler v1.1.1-0.20230831050635-df1bd5aae9df
+replace (
+	github.com/rancher/aks-operator v1.2.0 => github.com/KevinJoiner/rancher-aks-operator v0.0.0-20240124143758-627d80adf4e6
+	github.com/rancher/channelserver v0.5.1-0.20230719220800-0a37b73c7df8 => github.com/KevinJoiner/channelserver v0.0.0-20240124134223-b6d3819b0a98
+	github.com/rancher/eks-operator v1.3.0 => github.com/KevinJoiner/rancher-eks-operator v0.0.0-20240123183955-8f2f749b3ed1
+	github.com/rancher/gke-operator v1.2.0 => github.com/KevinJoiner/rancher-gke-operator v0.0.0-20240124144130-9c4adac1def8
```

Comment:
Remove and update commit when https://github.com/rancher/gke-operator/pull/297 is merged.

Reply:
It was merged, please rebase